templates/20-04/index.html

```diff
@@ -0,0 +1,287 @@
+      alt="",
+      width="2464",
+      height="1027",
+      hi_def=True,
+      loading="lazy"
+      ) | safe
+      }}
+    </div>
+  </section>
+
+  {%- call(slot) vf_tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=false) -%}
+    {%- if slot == 'title' -%}
+      <h2>What do you get with Ubuntu Pro for 20.04 LTS?</h2>
+    {%- endif -%}
+
+    {%- if slot == 'description' -%}
+      <div class="p-section--shallow">
+        <p>
+          Standard Security Maintenance of an Ubuntu LTS release covers binary packages that reside in the Main Ubuntu repository for a period of five years. For continued security beyond the standard five-year maintenance period, Ubuntu Pro delivers security maintenance to a wide range of binary packages that are commonly used in cloud and server workloads on 64-bit x86 AMD/Intel architectures for a period of five years beyond the end of standard support.
+        </p>
+        <p>Additionally, Ubuntu Pro can also cover all packages in the Ubuntu Universe repository, until April 2030.</p>
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_1' -%}
+      <h5>Access to Canonical partner portal</h5>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <p>
+        Resell the full range of Canonical's portfolio of private and public
+        cloud products: OpenStack, Kubernetes, IoT, support, security and
+        compliance for Ubuntu.
+      </p>
+      <table aria-label="Ubuntu releases and their associated architecture and kernel version">
+        <thead>
+          <tr>
+            <th>Ubuntu release</th>
+            <th>Architecture</th>
+            <th>Kernel Version</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th>
+              <strong>Ubuntu 18.04 LTS</strong>
+            </th>
+            <td>arm64, x86-64 (AMD/Intel) and s390x</td>
+            <td>5.4 (GA)</td>
+          </tr>
+          <tr>
+            <th>
+              <strong>Ubuntu 20.04 LTS</strong>
+            </th>
+            <td>arm64, x86-64  (AMD/Intel) and s390x</td>
+            <td>5.15 (HWE)</td>
+          </tr>
+        </tbody>
+      </table>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      <h5>Infrastructure packages</h5>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <p>
+        For Ubuntu 20.04 LTS, where technically feasible, Canonical provides extended security maintenance to all binary packages that reside in the Ubuntu Main Repository.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_3' -%}
+
+      <h5>Application packages</h5>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <p>
+        Canonical provides application extended security maintenance to the binary packages that reside in Main and Universe. A few commonly-used packages include:
```

Comment:
There should also be line break or a separate paragraph here as per design
```suggestion
        Canonical provides application extended security maintenance to the binary packages that reside in Main and Universe. <br />A few commonly-used packages include:
```